Added Page Number to UI

ItsDobes opened the pull request.

ItsDobes committed:
Added Page Number to UI

DanielMorin44 reviewed (commented):
Lets edit the HUD object prefab to include the Page Number text object instead of just the HUD object that exists in Chapter1.

ItsDobes committed:
Added Page Number Text to HUD prefab

ItsDobes committed:
Reconfigured Animations + Added Wallslide SFX

Wallsliding SFX is tricky because it's continuous and not one-off, so I had to change the way the animator was set up

ItsDobes committed:
Updated Page GUI

Page GUI now shows the page number of the page that's opened/visible instead of the active level when you're in the radial page selection menu

DanielMorin44 reviewed (approved):
Looks good, but we should start developing from discrete branches so that multiple features don't accidentally get bundled together.
